### PR TITLE
slack-config: add 1.22 RT Leads to the release-team-lead usergroup

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Adding 1.22 RT Leads to release-team-leads Slack usergroup.

Which issue(s) this PR fixes:

ref: https://github.com/kubernetes/sig-release/issues/1522

/assign
/sig release
/priority critical-urgent
/assign @justaugustus @saschagrunert @jeremyrickard @hasheddan
/cc @annajung @divya-mohan0209 @erismaster @kikisdeliveryservice

### CODE REVIEW

**Review by hasheddan — APPROVED** [review]
/lgtm

**Comment by jeremyrickard** [comment]
Should we retire this to reflect 1.22 instead of 1.21?

**Comment by hasheddan** [comment]
@jeremyrickard good catch :)

/retitle slack-config: add 1.22 RT Leads to the release-team-lead usergroup

**Comment by savitharaghunathan** [comment]
> Should we retitle this to reflect 1.22 instead of 1.21?

Thanks for the catch, @jeremyrickard. Copy-paste error 🤷‍♀️

**Comment by savitharaghunathan** [comment]
@hasheddan Fixed commit message to reflect 1.22 instead of 1.21. Can you `lgtm` again? Thanks :)

**Review by justaugustus — CHANGES_REQUESTED** [review]
@savitharaghunathan @guineveresaenger -- Should we add the EA to this group as well?
/hold

**Comment by savitharaghunathan** [comment]
> @savitharaghunathan @guineveresaenger -- Should we add the EA to this group as well?
> /hold

@justaugustus: Its a part of release team lead access issue, so I figured it is for the leads. I see no harm in adding an entry for EA.

File: communication/slack-config/sig-release/usergroups.yaml
Comment (justaugustus) [inline communication/slack-config/sig-release/usergroups.yaml:49]:
```suggestion
      - annajung # 1.22 Release Team Lead Shadow
```

Nitty nit

**Comment by justaugustus** [comment]
> I figured it is for the leads

@savitharaghunathan -- yep, primarily. We also include SIG Release leads here to make sure:
- you all have coverage for questions outside of scope for the RT
- SIG leads have visibility into "escalations"

Given the EA role is meant to share some of that responsibility, I think the add here makes sense.

File: communication/slack-config/sig-release/usergroups.yaml
Comment (savitharaghunathan) [reply communication/slack-config/sig-release/usergroups.yaml]: Fixed

File: communication/slack-config/users.yaml
Comment (jeefy) [inline communication/slack-config/users.yaml:10]: Same here -- can you move anna down below ankeesler so it's in alphabetical order? Thanks!

**Review by jeefy — CHANGES_REQUESTED** [review]
Couple quick fixes but otherwise it lgtm!

File: communication/slack-config/users.yaml
Comment (justaugustus) [reply communication/slack-config/users.yaml:10]: Alpha-sort strikes again!!

Comment (justaugustus) [reply communication/slack-config/users.yaml:10]: Fixed in https://github.com/kubernetes/community/pull/5745.

**Pull request merged**
